scripts/docker/cable.yml

```
@@ -0,0 +1,9 @@
+local: &local
+  adapter: redis
+  url: redis://localhost:6379
+
+production: *local
+development:
+  adapter: redis
+  url: redis://localhost:6379
```

This needs to be 
```suggestion
  url: redis://redis:6379
```
Since it's accessing the Redis server through the Docker network